docs/dissemination/deliverables.md

```diff
@@ -0,0 +1,15 @@
+# Deliverables
+
+Find below the deliverables released so far by the CRYPTECS consortium. In case
+you need access to deliverables classified as internal please get in touch
+(:material-email: [cryptecs-oss@googlegroups.com](mailto:cryptecs-oss@googlegroups.com)).
+
+## WP1 Project Coordination & Management
+
+**D1.1** Project Handbook (:octicons-lock-24: Internal)
+
+## WP2 Requirements Elicitation and Technical Analysis
+
+**D2.1** Use Case Specification ([:fontawesome-regular-file-pdf: .pdf][D2.1]{:target="_blank"})
+
+[D2.1]: https://drive.google.com/uc?export=download&id=1C5kpmS-E8S-RDBALxxgNw0O_7P37ods9
```

Comment:
I tried to open this link in a private window and it is not publicly accessible - is that intended?

Reply:
Hm. Tried also in a private tab on my side and from my private mobile phone. Both works. Are you sure?

Reply:
It works for me now.  Strange - I assume it was an issue on my end.